go.mod

```diff
@@ -10,9 +10,15 @@ require (
 	github.com/rs/zerolog v1.21.0
 	github.com/seabird-chat/seabird-go v0.4.0
 	golang.org/x/net v0.0.0-20210505214959-0714010a04ed // indirect
-	golang.org/x/oauth2 v0.0.0-20210427180440-81ed05c6b58c // indirect
-	golang.org/x/sys v0.0.0-20210503173754-0981d6026fa6 // indirect
+	golang.org/x/oauth2 v0.0.0-20210819190943-2bc19b11175f // indirect
+	golang.org/x/sys v0.0.0-20210510120138-977fb7262007 // indirect
 	google.golang.org/appengine v1.6.7 // indirect
-	google.golang.org/genproto v0.0.0-20210506142907-4a47615972c2 // indirect
-	google.golang.org/grpc v1.37.0 // indirect
+	google.golang.org/genproto v0.0.0-20210903162649-d08c68adba83 // indirect
+	google.golang.org/grpc v1.40.0 // indirect
+)
+
+require (
```

Comment:
I like how they're moving indirect deps to a separate block, but I wish they migrated all of them

Reply:
Hmm good point, I can just do that manually, interesting that they didn't do that, is there a reason you're aware of?

Reply:
Not that I'm aware of